content/en/docs/advanced-tasks/overriding-catalog-source-pod-scheduling-configuration.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Overriding CatalogSource Pod Scheduling Configuration"
+linkTitle: "CatalogSource Pod Scheduling Overrides"
+weight: 3
+---
+
+## Overriding CatalogSource Pod Scheduling Configuration
```

Comment:
Do we need to mention the OLM release this functionality was added (assuming we cut a new OLM minor version release soon), or can we just rely on the olm-docs releases for this instead?

Reply:
I currently don't know how documetation works downstream =S

Reply:
@kevinrizza any ideas here?

Reply:
the docs should represent the current state in master, and we cut docs releases for different minor versions of OLM